jujugui/static/gui/src/app/store/env/base.js

```diff
@@ -469,14 +469,14 @@ YUI.add('juju-env-base', function(Y) {
     },
 
     /**
-     * Store the user's credentials in session storage.
-     *
-     * @method setCredentials
-     * @param {Object} The credentials to store, with a "user" and a "password"
-     *   attribute included, or with a "macaroons" attribute, depending on the
-     *   method used to log in. The user must be a user name, not a user tag.
-     * @return {undefined} Stores data only.
-     */
+      Store the user's credentials in session storage.
+
+      @method setCredentials
+      @param {Object} The credentials to store.
+        Possible properties { user, password, macaroons, external }.
```

Comment:
Please mention that user and password must be strings, macaroons is a list of objects and external is an object.